### ISSUE TITLE
Anaconda Navigator crashes after showing main window (Windows 10)

### ISSUE THREAD

**Issue by ahnugent** [ContinuumIO/anaconda-issues#4808]
I just updated Anaconda Navigator (hoping it could finally render the launcher at a readable resolution) and now it crashes when I try to run it. It gets as far as displaying the main window (happily, at a useful resolution) then closes after about one second.

I found similar issues elsewhere in this repo and tried to follow the advice to remove the ".continuum" folder, but there doesn't seem to be one. (I've looked in "C:\Users\[myname]\Anaconda3" and "C:\Program Files\Anaconda3".)

I ran the update like this:
`conda update anaconda-navigator`

I am using Windows 10 64-bit.

**Comment by goanpeca** [comment]
@ahnugent open up the anaconda prompt and type `$ anaconda-navigator` and paste any traceback so we debug the problem.

**Comment by ahnugent** [comment]
Thanks, here it is:

```
(C:\Users\allen\Anaconda3) C:\Users\allen>anaconda-navigator
QWindowsFontDatabase::createEngine: CreateFontFromLOGFONT failed ()
QWindowsFontDatabase::createEngine: CreateFontFromLOGFONT failed ()
QWindowsFontDatabase::createEngine: CreateFontFromLOGFONT failed ()
QWindowsFontDatabase::createEngine: CreateFontFromLOGFONT failed ()
QWindowsFontDatabase::createEngine: CreateFontFromLOGFONT failed ()
QWindowsFontDatabase::createEngine: CreateFontFromLOGFONT failed ()
QWindowsFontDatabase::createEngine: CreateFontFromLOGFONT failed ()
QWindowsFontDatabase::createEngine: CreateFontFromLOGFONT failed ()
QWindowsFontDatabase::createEngine: CreateFontFromLOGFONT failed ()
QWindowsFontDatabase::createEngine: CreateFontFromLOGFONT failed ()
Traceback (most recent call last):
  File "C:\Users\allen\Anaconda3\lib\site-packages\anaconda_navigator\widgets\tabs\community.py", line 372, in set_content_list
    uri=item['uri'], date=item.get('date', '') or "",
KeyError: 'uri'

(C:\Users\allen\Anaconda3) C:\Users\allen>
```

**Comment by goanpeca** [comment]
@ahnugent seems you are using an old version of navigator. Please update to the latest version. On the anaconda prompt enter:

```
$ conda update anaconda-navigator
$ conda install navigator-updater
```

**Comment by ahnugent** [comment]
When I tried `conda update anaconda-navigator` yesterday, it downgraded my version. I just followed your instructions but I have the same problem.

**Comment by ahnugent** [comment]
Correction: 
Using the Windows shortcut I get the old problem back. If I run navigator from the Anaconda Prompt, the problem goes away although the traceback shows some other issue(s):

```
C:\WINDOWS\system32>anaconda-navigator
the JSON object must be str, not 'bytes'
the JSON object must be str, not 'bytes'
2017-09-01 12:21:05,885 - ERROR download_api._download:197
Invalid url http://content.continuum.io/api/events

2017-09-01 12:21:06,014 - ERROR download_api._download:197
Invalid url http://content.continuum.io/api/videos

2017-09-01 12:21:06,415 - ERROR download_api._download:197
Invalid url http://content.continuum.io/api/webinars

2017-09-01 12:21:06,424 - ERROR community.load_content:203
('C://Users//allen//.anaconda//navigator//content//webinars.json', JSONDecodeError('Unterminated string starting at: line 1 column 43387 (char 43386)',))

2017-09-01 12:21:14,872 - ERROR download_api._download:192
Invalid URL '': No schema supplied. Perhaps you meant http://?

2017-09-01 12:21:14,880 - ERROR download_api._download:197
Invalid url

2017-09-01 12:21:19,682 - ERROR download_api._download:192
Invalid URL '': No schema supplied. Perhaps you meant http://?

2017-09-01 12:21:19,688 - ERROR download_api._download:197
Invalid url
```

**Comment by ahnugent** [comment]
I'll see how I go. 

Thanks!

**Comment by goanpeca** [comment]
@ahnugent not sure why the shortcut is not working, in the meantime please run from the command prompt. The issue you are seeing are currently being fixed and will be available next week with 1.6.6.

**Comment by venkatshukla** [comment]
Windows+s --> search for Anaconda Navigator --> Right Click and Run as Administrator.. This is working for me After Updating Anaconda.

**Comment by MarkMoretto** [comment]
@goanpeca  Thanks for the resolution.  I was having the same problem and this worked for me.

Usually I just launch Spyder directly and just updated everything (to ver. 3.2.2, I believe) and the crash might have happened after that for me.

**Comment by goanpeca** [comment]
@venkatshukla 

>Windows+s --> search for Anaconda Navigator --> Right Click and Run as Administrator.. This is working for me After Updating Anaconda.

If you do this now you will always need to run as admin. We do not recommend all user installs. The current user install is always recommended.

**Comment by mayatp** [comment]
Thanks @goanpeca!
`$ conda update anaconda-navigator
$ conda install navigator-updater`
fixed the issue for me as well.

**Comment by Raulster24** [comment]
Thanks @goanpeca!
$ conda update anaconda-navigator
$ conda install navigator-updater
fixed the issue for me as well.
I ran the anaconda prompt with administrator privilege and was able to update with above commands.

**Comment by MarcDecosta** [comment]
$ conda update anaconda-navigator
$ conda install navigator-updater

Run above two codes in CMD as administrator.
Run the navigator as administrator.

**Comment by arnab2183** [comment]
I am finding a unique issue whiling launching the anaconda navigator. its getting launched, however post 2 to 3 seconds its getting closed on its own. can you help me in any way regarding the same. I am very new in using the platform. it will be of great help if you can assist me.

**Comment by Raulster24** [comment]
Hi @arnab2183 ,
Have you updated your anaconda version, i also faced the same issue but on updating it is working fine now. Please follow the steps mentioned above to update it.
Also try to run as administrator whenever you start anaconda!

**Comment by pavanjava** [comment]
$ conda update anaconda-navigator
$ conda install navigator-updater

Fixed my Issue

**Comment by arnab2183** [comment]
I am finding a error whiling running the code in Anaconda prompt. Can any
one help.

**Comment by Raulster24** [comment]
Hey @arnab2183 , Can you post the snapshot or give more detail as in what error you are getting?

**Comment by MarcDecosta** [comment]
@arnab2183 
Run the cmd as administrator.
Go to the installation folder using cd command.
Then run the below two commands separately.
 conda update anaconda-navigator
 conda install navigator-updater
It will do the job.

**Comment by ELayneMorrison** [comment]
Marc, I'm having this error too. I'm on Windows 7 running Command Prompt as admin: 'conda is not recognized as an internal or external command' (same  from Anaconda Prompt) Any ideas?

**Comment by learninguser** [comment]
@MarcDecosta Please make sure that "...\Anaconda3\Script" is in the path variable of your system. You might have forgotten to select the option to add this to path variable while installing anaconda.

Solution 2: Please start anaconda prompt from the search bar and perform the update using the steps mentioned above. This should work as it worked for me. Hope this helps you.

**Comment by pavanjava** [comment]
Hi @arnab2183 & @ELayneMorrison can you please run the below commands. i too had the same problem but after running the below commands it fixed my problem.

$ conda update anaconda-navigator
$ conda install navigator-updater

**Comment by Fakhimi** [comment]
Hi, I tried all of the above solutions, but, unfortunately, none of them worked for me. 
I have a win7 (32 bit). When I open the anaconda navigator, it opens prompt and suddenly closes it.
My python version is 3.6 and my anaconda version is 3.5.1.

In fact, I tried these codes in anaconda prompt:

$ conda update anaconda-navigator
$ conda install navigator-updater

**Comment by goanpeca** [comment]
@Fakhimi please open the Anaconda Prompt and run the following code:

```
conda info
```
and

```
conda list
```

**Comment by Fakhimi** [comment]
It's the result of "conda info"
![1](https://user-images.githubusercontent.com/37240371/37246197-13a25fa4-24ba-11e8-8e92-f10b8b67345a.png)

**Comment by vaibhavs21** [comment]
Hello guys,
My anaconda navigator crashes only after a sec after the launch.
I tried updating the navigator and it shows the anaconda navigator is already up to date.
Still the issue persists.

Can somebody help.
Thanks in advance.

**Comment by CarolineMorton** [comment]
Same issue here... It looks like it is opening and then immediately shuts. I have updated as per instructions above and restarted everything but no change. Any ideas?

**Comment by goanpeca** [comment]
@vaibhavs21 @CarolineMorton please open a new issue. This issue is closed and if the solutions posted here don't solve the problem then it is a different thing.

**Comment by Adorra** [comment]
@vaibhavs21 @CarolineMorton I'm in a similar situation with the above solutions that have been applied. Anaconda Navigator states it is already up to date.

**Comment by asutosh405** [comment]
running the 2 commands mentioned in the thread solved the purpose for me.
I was also facing the same issue.

thumbs up .. !!!

**Comment by NinaDes** [comment]
Hi, I got the same error.
I tryed launching
$ conda update conda
$ conda update anaconda-navigator
$ conda install navigator-updater

then I also launched 
$ anaconda-navigator --reset as administrator but I still don't understand how to solve the problem.



here is my conda info

     active environment : None
       user config file : C:\Users\n.desgranges\.condarc
 populated config files : C:\Users\n.desgranges\.condarc
          conda version : 4.5.5
    conda-build version : 2.0.2
         python version : 2.7.12.final.0
       base environment : C:\Users\n.desgranges\AppData\Local\Continuum\Anaconda2  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : C:\Users\n.desgranges\AppData\Local\Continuum\Anaconda2\pkgs
                          C:\Users\n.desgranges\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\n.desgranges\AppData\Local\Continuum\Anaconda2\envs
                          C:\Users\n.desgranges\AppData\Local\conda\conda\envs
                          C:\Users\n.desgranges\.conda\envs
               platform : win-64
             user-agent : conda/4.5.5 requests/2.9.1 CPython/2.7.12 Windows/10 Windows/10.0.16299
          administrator : True
             netrc file : None
           offline mode : False


and my conda list

_license                  1.1                      py27_1  
_nb_ext_conf              0.3.0                    py27_0  
alabaster                 0.7.9                    py27_0    conda-forge
anaconda                  custom                   py27_0    anaconda
anaconda-clean            1.0.0                    py27_0  
anaconda-client           1.5.1                    py27_0    conda-forge
anaconda-navigator        1.3.1                    py27_0  
appdirs                   1.4.2                     <pip>
argcomplete               1.0.0                    py27_1  
aspy.yaml                 1.1.1                     <pip>
astroid                   1.4.8                    py27_0    conda-forge
astropy                   2.0.1               np112py27_0    anaconda
awscli                    1.6.2                     <pip>
babel                     2.3.4                    py27_0    conda-forge
backports                 1.0                      py27_0  
backports.functools-lru-cache 1.5                       <pip>
backports.functools_lru_cache 1.2.1                    py27_0    conda-forge
backports.shutil_get_terminal_size 1.0.0                    py27_0    conda-forge
backports_abc             0.5                      py27_0    conda-forge
bcdoc                     0.12.2                    <pip>
beautifulsoup4            4.5.1                    py27_0  
bitarray                  0.8.1                    py27_1  
blaze                     0.10.1                   py27_0  
bokeh                     0.12.2                   py27_0  
boto                      2.33.0                    <pip>
boto                      2.42.0                   py27_0  
botocore                  0.73.0                    <pip>
bottleneck                1.2.1               np112py27_0    anaconda
bz2file                   0.98                     py27_0    anaconda
bzip2                     1.0.6                     vc9_1  [vc9]  conda-forge
ca-certificates           2018.4.16                     0    conda-forge
cached-property           1.4.2                     <pip>
cdecimal                  2.3                      py27_2  
certifi                   2015.04.28                <pip>
certifi                   2018.4.16                py27_0    conda-forge
cffi                      1.7.0                    py27_0  
cfgv                      1.1.0                     <pip>
chest                     0.2.3                    py27_0  
click                     6.6                      py27_1    conda-forge
cloudpickle               0.2.1                    py27_2    conda-forge
clyent                    1.2.1                    py27_0    conda-forge
colorama                  0.2.5                     <pip>
colorama                  0.3.7                    py27_0    conda-forge
comtypes                  1.1.2                    py27_0  
conda                     4.5.5                    py27_0    conda-forge
conda-build               2.0.2                    py27_0  
conda-env                 2.6.0                         0    conda-forge
configobj                 5.0.6                    py27_0  
configparser              3.5.0                    py27_0    conda-forge
console_shortcut          0.1.1                    py27_1  
contextlib2               0.5.4                    py27_0    conda-forge
cryptography              1.5                      py27_0  
curl                      7.49.1                    vc9_2  [vc9]  conda-forge
cycler                    0.10.0                   py27_0    conda-forge
cymem                     1.31.2               py27_vc9_0  [vc9]  conda-forge
cytoolz                   0.8.2                    py27_0    conda-forge
dask                      0.10.0                   py27_0    conda-forge
datashape                 0.5.2                    py27_0  
decorator                 4.0.10                   py27_0    conda-forge
decorator                 4.0.9                     <pip>
dill                      0.2.6                     <pip>
dill                      0.2.5                    py27_1    conda-forge
docutils                  0.12                     py27_0    conda-forge
ediblepickle              1.1.3                     <pip>
elephas                   0.3                       <pip>
entrypoints               0.2.2                     <pip>
entrypoints               0.2.2                    py27_0    conda-forge
enum34                    1.1.6                    py27_1    conda-forge
et_xmlfile                1.0.1                    py27_0  
fastcache                 1.0.2                    py27_1  
filelock                  2.0.6                    py27_0    conda-forge
flask                     0.11.1                   py27_0    conda-forge
flask-cors                3.0.2                    py27_0    conda-forge
freetype                  2.7                       vc9_1  [vc9]  conda-forge
ftfy                      4.4.2                    py27_0    conda-forge
funcsigs                  1.0.2                    py27_0    conda-forge
functools32               3.2.3.2                  py27_1    conda-forge
future                    0.16.0                   py27_0    anaconda
futures                   3.0.5                    py27_0    conda-forge
gapic-google-cloud-datastore-v1 0.15.1                    <pip>
gapic-google-cloud-error-reporting-v1beta1 0.15.0                    <pip>
gapic-google-cloud-logging-v2 0.91.0                    <pip>
gapic-google-cloud-pubsub-v1 0.15.0                    <pip>
gapic-google-cloud-spanner-admin-database-v1 0.15.0                    <pip>
gapic-google-cloud-spanner-admin-instance-v1 0.15.0                    <pip>
gapic-google-cloud-spanner-v1 0.15.0                    <pip>
gapic-google-cloud-speech-v1beta1 0.15.2                    <pip>
gapic-google-cloud-vision-v1 0.15.2                    <pip>
gensim                    2.3.0                py27_vc9_0  [vc9]  conda-forge
genson                    0.1.0                     <pip>
geopy                     1.11.0                   py27_0    conda-forge
get_terminal_size         1.0.0                    py27_0  
gevent                    1.1.2                    py27_0  
google-api-python-client  1.6.2                     <pip>
google-api-python-client  1.5.1                    py27_0    conda-forge
google-auth               0.8.0                     <pip>
google-auth-httplib2      0.0.2                     <pip>
google-cloud              0.23.0                    <pip>
google-cloud-bigquery     0.23.0                    <pip>
google-cloud-bigtable     0.23.0                    <pip>
google-cloud-core         0.23.1                    <pip>
google-cloud-datastore    0.23.0                    <pip>
google-cloud-dns          0.23.0                    <pip>
google-cloud-error-reporting 0.23.0                    <pip>
google-cloud-language     0.23.0                    <pip>
google-cloud-logging      0.23.1                    <pip>
google-cloud-monitoring   0.23.0                    <pip>
google-cloud-pubsub       0.23.0                    <pip>
google-cloud-resource-manager 0.23.0                    <pip>
google-cloud-runtimeconfig 0.23.0                    <pip>
google-cloud-spanner      0.23.1                    <pip>
google-cloud-speech       0.23.0                    <pip>
google-cloud-storage      0.23.0                    <pip>
google-cloud-translate    0.23.0                    <pip>
google-cloud-vision       0.23.2                    <pip>
google-gax                0.15.6                    <pip>
googleapis-common-protos  1.5.2                     <pip>
goslate                   1.5.1                     <pip>
graphviz                  0.7.1                     <pip>
graphviz                  2.38.0                        4    anaconda
greenlet                  0.4.10                   py27_0  
grin                      1.2.1                    py27_3  
grpc-google-iam-v1        0.11.1                    <pip>
grpcio                    1.1.3                     <pip>
h2o                       3.10.0.9                      0    anaconda
h2o-py                    3.10.0.9                 py27_0    anaconda
h5py                      2.7.0               np112py27_0    conda-forge
hdf5                      1.8.17                    vc9_8  [vc9]  conda-forge
heapdict                  1.0.0                    py27_1  
html5lib                  0.999999999              py27_0    conda-forge
httplib2                  0.10.3                   py27_0    conda-forge
httplib2                  0.10.3                    <pip>
hyperas                   0.3                       <pip>
hyperopt                  0.1                       <pip>
icu                       58.2                      vc9_0  [vc9]  conda-forge
identify                  1.0.18                    <pip>
idna                      2.1                      py27_0    conda-forge
imagesize                 0.7.1                    py27_0    conda-forge
ipaddress                 1.0.16                   py27_0    conda-forge
ipykernel                 4.5.2                    py27_0    conda-forge
ipython                   5.1.0                    py27_1    conda-forge
ipython-sql               0.3.8                     <pip>
ipython_genutils          0.1.0                    py27_0    conda-forge
ipywidgets                5.2.2                    py27_2    conda-forge
isort                     4.2.5                    py27_0    conda-forge
itsdangerous              0.24                     py27_0  
jdcal                     1.2                      py27_1  
jedi                      0.9.0                    py27_1  
jinja2                    2.8                      py27_1    conda-forge
jmespath                  0.5.0                     <pip>
jpeg                      9b                        vc9_2  [vc9]  conda-forge
jsonschema                2.5.1                    py27_0    conda-forge
jupyter                   1.0.0                    py27_3  
jupyter-contrib-core      0.3.3                     <pip>
jupyter-contrib-nbextensions 0.5.0                     <pip>
jupyter-highlight-selected-word 0.2.0                     <pip>
jupyter-latex-envs        1.3.8.4                   <pip>
jupyter-nbextensions-configurator 0.4.0                     <pip>
jupyter_client            4.4.0                    py27_0    conda-forge
jupyter_console           5.0.0                    py27_0    conda-forge
jupyter_contrib_core      0.3.0                    py27_0    conda-forge
jupyter_core              4.2.0                    py27_0    conda-forge
jupyter_dashboards        0.6.1                    py27_0    conda-forge
jupyter_nbextensions_configurator 0.2.3                    py27_0    conda-forge
Keras                     1.1.2                     <pip>
kiwisolver                1.0.1                     <pip>
langdetect                1.0.7                      py_0    conda-forge
lazy-object-proxy         1.2.2                    py27_0    conda-forge
ldap3                     2.2.4                    py27_0    conda-forge
libiconv                  1.14                      vc9_4  [vc9]  conda-forge
libpng                    1.6.30                    vc9_1  [vc9]  anaconda
libpython                 2.0                      py27_0    anaconda
libsodium                 1.0.16                    vc9_0  [vc9]  conda-forge
libtiff                   4.0.7                     vc9_0  [vc9]  conda-forge
libxml2                   2.9.4                     vc9_2  [vc9]  conda-forge
libxslt                   1.1.29                    vc9_1  [vc9]  conda-forge
llvmlite                  0.19.0                   py27_0    anaconda
locket                    0.2.0                    py27_0    conda-forge
lxml                      4.1.0                     <pip>
lxml                      3.6.4                    py27_0    conda-forge
markupsafe                0.23                     py27_0    conda-forge
matplotlib                2.0.2               np112py27_0    anaconda
matplotlib                2.2.2                     <pip>
mccabe                    0.5.0                    py27_0    conda-forge
menuinst                  1.4.1                    py27_0  
mingw                     4.7                           1    anaconda
mistune                   0.7.3                    py27_0    conda-forge
mkl                       2017.0.3                      0    anaconda
mkl-service               1.1.2                    py27_2  
mlxtend                   0.7.0                    py27_0    conda-forge
mpmath                    0.19                     py27_0    conda-forge
multipledispatch          0.4.8                    py27_0    conda-forge
murmurhash                0.26.4               py27_vc9_0  [vc9]  conda-forge
navigator-updater         0.2.1                    py27_0    anaconda
nb_anacondacloud          1.2.0                    py27_0    conda-forge
nb_conda                  2.0.0                    py27_0    conda-forge
nb_conda_kernels          2.0.0                    py27_0    conda-forge
nbconvert                 4.2.0                    py27_0    conda-forge
nbformat                  4.2.0                    py27_0    conda-forge
nbpresent                 3.0.2                    py27_1    conda-forge
networkx                  1.11                     py27_0    conda-forge
nltk                      3.2.1                    py27_0    conda-forge
nodeenv                   1.3.0                     <pip>
nose                      1.3.7                    py27_2    conda-forge
notebook                  4.2.3                    py27_0    conda-forge
numba                     0.34.0              np112py27_0    anaconda
numexpr                   2.6.2               np112py27_0    anaconda
numpy                     1.14.4                    <pip>
numpy                     1.12.1                   py27_0    anaconda
oauth2client              3.0.0                     <pip>
oauth2client              2.2.0                    py27_0    conda-forge
odo                       0.5.0                    py27_1  
olefile                   0.44                     py27_0    conda-forge
openjdk                   8.17.0.3                      1    anaconda
openpyxl                  2.3.2                    py27_0  
openssl                   1.0.2o                    vc9_0  [vc9]  conda-forge
packaging                 16.8                      <pip>
pandas                    0.20.3                   py27_0    anaconda
partd                     0.3.6                    py27_0    conda-forge
path.py                   8.2.1                    py27_0    conda-forge
pathlib                   1.0.1                    py27_0    conda-forge
pathlib2                  2.1.0                    py27_0    conda-forge
patsy                     0.4.1                    py27_0    conda-forge
pbr                       3.1.1                     <pip>
pep8                      1.7.0                    py27_0    conda-forge
pickleshare               0.7.3                    py27_0    conda-forge
pillow                    4.3.0                    py27_0    conda-forge
pip                       10.0.1                    <pip>
pip                       9.0.1                    py27_0    conda-forge
pkginfo                   1.2.1                    py27_0    conda-forge
plac                      0.9.6                    py27_0  
pluggy                    0.6.0                     <pip>
ply                       3.8                       <pip>
ply                       3.9                      py27_0  
pre-commit                1.10.1                    <pip>
preshed                   1.0.0                py27_vc9_0  [vc9]  conda-forge
prettytable               0.7.2                     <pip>
prompt_toolkit            1.0.9                    py27_0    conda-forge
proto-google-cloud-datastore-v1 0.90.1                    <pip>
proto-google-cloud-error-reporting-v1beta1 0.15.1                    <pip>
proto-google-cloud-logging-v2 0.91.0                    <pip>
proto-google-cloud-pubsub-v1 0.15.1                    <pip>
proto-google-cloud-spanner-admin-database-v1 0.15.1                    <pip>
proto-google-cloud-spanner-admin-instance-v1 0.15.1                    <pip>
proto-google-cloud-spanner-v1 0.15.1                    <pip>
proto-google-cloud-speech-v1beta1 0.15.1                    <pip>
proto-google-cloud-vision-v1 0.15.2                    <pip>
protobuf                  3.2.0                     <pip>
psclient                  2.0                       <pip>
psutil                    4.3.0                    py27_0    conda-forge
py                        1.4.31                   py27_0    conda-forge
py-translate              1.0.3                     <pip>
pyasn1                    0.1.9                    py27_0    conda-forge
pyasn1                    0.2.3                     <pip>
pyasn1-modules            0.0.8                     <pip>
pyasn1-modules            0.0.8                    py27_0    conda-forge
pycosat                   0.6.3                    py27_0    conda-forge
pycparser                 2.17                     py27_0    conda-forge
pycrypto                  2.6.1                    py27_0    conda-forge
pycurl                    7.43.0                   py27_0  
pydotplus                 2.0.2                    py27_0    conda-forge
pyflakes                  1.3.0                    py27_0    conda-forge
pygments                  2.1.3                    py27_1    conda-forge
pylint                    1.6.4                    py27_0    conda-forge
pymongo                   3.4.0                     <pip>
pyneurgen                 0.3.1                     <pip>
pyopenssl                 16.2.0                   py27_0    anaconda
pyparsing                 2.1.10                   py27_0    conda-forge
pyparsing                 2.2.0                     <pip>
pyqt                      5.6.0                    py27_4    conda-forge
pyreadline                2.1                      py27_0    conda-forge
pytables                  3.4.2               np112py27_0    anaconda
pytest                    3.0.5                    py27_0    conda-forge
python                    2.7.12                        1    conda-forge
python-crfsuite           0.9.2                py27_vc9_0  [vc9]  conda-forge
python-dateutil           2.7.3                     <pip>
python-dateutil           2.6.0                    py27_0    conda-forge
pytz                      2016.7                   py27_0    conda-forge
pytz                      2018.4                    <pip>
pywavelets                0.5.2               np112py27_0    anaconda
pywin32                   220                      py27_1  
pyyaml                    3.11                     py27_0    conda-forge
pyzmq                     16.0.2                   py27_0    conda-forge
qt                        5.6.2                     vc9_2  [vc9]  conda-forge
qtawesome                 0.3.3                    py27_0    conda-forge
qtconsole                 4.2.1                    py27_0    conda-forge
qtpy                      1.4.2                    py27_0    anaconda
regex                     2017.07.28               py27_0    conda-forge
requests                  2.12.4                   py27_0    conda-forge
requests                  2.9.1                     <pip>
rope                      0.9.4                    py27_1  
rsa                       3.4.2                     <pip>
rsa                       3.4.2                    py27_0    conda-forge
rsa                       3.1.2                     <pip>
ruamel_yaml               0.11.14                  py27_0    anaconda
scikit-image              0.13.0              np112py27_0    anaconda
scikit-learn              0.19.0              np112py27_0    anaconda
scikit-surprise           1.0.4            py27h894d8be_0    nicolashug
scipy                     0.19.1              np112py27_0    anaconda
seaborn                   0.7.1                    py27_0    anaconda
setuptools                39.2.0                   py27_0    conda-forge
setuptools                39.2.0                    <pip>
simplegeneric             0.8.1                    py27_0    conda-forge
simplejson                3.3.0                    py27_0    asmeurer
singledispatch            3.4.0.3                  py27_0    conda-forge
sip                       4.18                     py27_0    conda-forge
six                       1.11.0                    <pip>
six                       1.10.0                   py27_1    conda-forge
sklearn                   0.0                       <pip>
smart_open                1.5.3                    py27_0    anaconda
snowballstemmer           1.2.0                    py27_0    salford_systems
sockjs-tornado            1.0.3                    py27_0  
spacy                     1.9.0           np112py27_vc9_1  [vc9]  conda-forge
sphinx                    1.5                      py27_0    conda-forge
spyder                    2.3.9                    py27_1    conda-forge
sqlalchemy                1.0.13                   py27_0  
sqlite                    3.13.0                    vc9_1  [vc9]  conda-forge
sqlparse                  0.2.4                     <pip>
ssl_match_hostname        3.5.0.1                  py27_0    conda-forge
statsmodels               0.8.0               np112py27_0    anaconda
stop-words                2015.2.23.1               <pip>
subprocess32              3.2.7                    py27_0    anaconda
sympy                     1.0                      py27_0    conda-forge
tabulate                  0.7.7                    py27_0    conda-forge
termcolor                 1.1.0                    py27_1    conda-forge
Theano                    0.8.2                     <pip>
thinc                     6.5.2           np112py27_vc9_0  [vc9]  conda-forge
tk                        8.5.19                    vc9_1  [vc9]  conda-forge
toml                      0.9.4                     <pip>
toolz                     0.8.2                    py27_0    conda-forge
tornado                   4.4.2                    py27_0    conda-forge
tornado                   4.3                       <pip>
tox                       3.0.0                     <pip>
tqdm                      4.15.0                   py27_0    conda-forge
traitlets                 4.3.1                    py27_0    conda-forge
translate                 3.5.0                     <pip>
translation               1.0.5                     <pip>
treetaggerwrapper         2.2.4                     <pip>
ujson                     1.35                     py27_0  
unicodecsv                0.14.1                   py27_0  
Unidecode                 0.04.20                   <pip>
uritemplate               0.6                      py27_0    conda-forge
uritemplate               3.0.0                     <pip>
vc                        9                             0    conda-forge
virtualenv                15.1.0                    <pip>
vs2008_runtime            9.00.30729.1                  2  
vs2015_runtime            14.0.25420                    0    conda-forge
wcwidth                   0.1.7                    py27_0    conda-forge
webencodings              0.5                      py27_0    conda-forge
werkzeug                  0.11.10                  py27_0    conda-forge
wheel                     0.29.0                   py27_0    conda-forge
widgetsnbextension        1.2.6                    py27_3    conda-forge
win_unicode_console       0.5                      py27_0    conda-forge
wincertstore              0.2                      py27_0    conda-forge
wordcloud                 1.2.1                    py27_0    conda-forge
wrapt                     1.10.8                   py27_0    conda-forge
xlrd                      1.0.0                    py27_0  
xlsxwriter                0.9.3                    py27_0  
xlwings                   0.10.0                   py27_0  
xlwt                      1.1.2                    py27_0  
xmltodict                 0.10.2                   py27_0    conda-forge
yaml                      0.1.6                     vc9_0  [vc9]  conda-forge
youtube-dl                2018.6.4                  <pip>
zeromq                    4.2.5                     vc9_1  [vc9]  conda-forge
zlib                      1.2.8                     vc9_3  [vc9]  conda-forge

**Comment by ypjoshi18** [comment]
> @ahnugent open up the anaconda prompt and type `$ anaconda-navigator` and paste any traceback so we debug the problem.

the prompt displayed nothing ... so what is the issue ?
